src/main/java/redis/clients/jedis/StreamEntryID.java

```diff
@@ -129,6 +147,7 @@ public String toString() {
   /**
    * Can be used in XRANGE, XREVRANGE and XPENDING commands.
    */
+  // TODO: FIRST_ID ?
```

Comment:
I would vote for keeping `MINIMUM_ID`, i.e. preserve the naming from Redis, as much as possible. Same for `MAXIMUM_ID`.